runner/runner.go

```diff
@@ -175,7 +175,11 @@ func (r *Runner) validate() error {
 
 // Cluster executes cluster runner defined in config.
 func (r *Runner) Cluster(ctx context.Context, list bool, selectiveChecks ...string) (*CombinedResponse, error) {
-	return r.run(ctx, r.ClusterChecks, list, selectiveChecks)
+	var clusterChecks []string
```

Comment:
I think we should probably have a method for this, something like 

```
r.clusterCheckNames() []string
```
That would make this method read a little bit easier.